### PR TITLE
Add 0.19.3-beta release notes

### PR DESCRIPTION
**Pull request by sgrebnov**
## 🗣 Description

Add 0.19.3-beta release notes

### CODE REVIEW

File: docs/release_notes/v0.19.3-beta.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+Spice v0.19.3-beta introduces the new `localpod` Data Connector, enhanced performance for federated queries across multiple sources, and improved stability of data connectors and accelerators.
+
+## Highlights in v0.19.3
+
+**Improved federated queries containing multiple sources**: Faster queries across multiple federated sources by optimizing when filters are applied.
+
+**Postgres Schema** Improved PostgreSQL Data Connector schema extraction supporting a wider range of column types and empty tables
+
+**localpod connector**: Implement "tiered" acceleration pattern by using 'localpod' Data Connector pointing to existing Spice dataset. By using `localpod` the data is available for nultiple datasets with different configuration while pulled only once.
```
Comment (eadgbear) [inline docs/release_notes/v0.19.3-beta.md]: Should we link to the `localpod` docs here?

File: docs/release_notes/v0.19.3-beta.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+- @slyons
+
+## New Contributors
+
+- @nlamirault made their first contribution in https://github.com/spiceai/spiceai/pull/3207
+- @barracudarin made their first contribution in https://github.com/spiceai/spiceai/pull/3228
+
+## What's Changed
+
+- Make Anthropic OpenAI compatible. by @Jeadie in https://github.com/spiceai/spiceai/pull/3087
```
Comment (slyons) [inline docs/release_notes/v0.19.3-beta.md:34]: Might want to mention this as a highlight, IMO

File: docs/release_notes/v0.19.3-beta.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+## Highlights in v0.19.3
+
+**Improved federated queries containing multiple sources**: Faster queries across multiple federated sources by optimizing when filters are applied.
+
+**Postgres Schema** Improved PostgreSQL Data Connector schema extraction supporting a wider range of column types and empty tables
+
+**localpod connector**: Implement "tiered" acceleration pattern by using 'localpod' Data Connector pointing to existing Spice dataset. By using `localpod` the data is available for nultiple datasets with different configuration while pulled only once.
+
+**DataFusion Fixes**: Resolved bugs in DataFusion and DataFusion Table Providers, improving TPC-DS query support and correctness.
```
Comment (lukekim) [inline docs/release_notes/v0.19.3-beta.md]: Add Anthropic support?

File: docs/release_notes/v0.19.3-beta.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
+# Spice v0.19.3 (Oct 28, 2024)
+
+Spice v0.19.3-beta introduces the new `localpod` Data Connector, enhanced performance for federated queries across multiple sources, and improved stability of data connectors and accelerators.
+
+## Highlights in v0.19.3
+
```
Comment (peasee) [inline docs/release_notes/v0.19.3-beta.md:6]:
```suggestion

**GitHub Connector Beta Release:** The GitHub Connector has graduated to Beta after extensive testing, and stability and performance improvements.

```

**Review by eadgbear — APPROVED** [review]

**Pull request merged**